examples/pytorch/TranslationFairseq/README.md

```diff
@@ -0,0 +1,62 @@
+## Fairseq en to de translation example with MLflow
+
+This example illustrates the training and testing process of English to German translation model using [fairseq](https://ai.facebook.com/tools/fairseq/) and its integration with MLflow.
+During the process, all the parameters, metrics, summary and the model were dumped into mlflow. 
+
+### Package Requirements
+
+Install the required packages mentioned in the following fairseq readme link
+
+https://pytorch.org/hub/pytorch_fairseq_translation/
+
+In addition to these, following pip packages are used
+
+1. prettytable
+2. ConfigParser
+3. mlflow
+4. pyximport
+
+### Clone the repository
+
+Follow the `Requirements and Installation` section from the following link to clone the repository
```

Comment:
@ankan94 Thanks for the PR, I was wondering if it might more straight forward to mention the command and git url.